src/operator/convolution-inl.h

```diff
@@ -97,6 +98,8 @@ struct ConvolutionParam : public dmlc::Parameter<ConvolutionParam> {
     .set_default(dmlc::optional<int>())
     .describe("Set layout for input, output and weight. Empty for\n    "
               "default layout: NCW for 1d, NCHW for 2d and NCDHW for 3d.");
+    DMLC_DECLARE_FIELD(depthwise_conv_off).set_default(false)
+    .describe("whether to turn off depthwise convolution for this layer");
```

Comment:
Let's remove this if there are no important reasons. Convolution has too many switches.